apps/zotonic_mod_admin/priv/templates/_admin_edit_content_page_connections_list.tpl

```diff
@@ -56,6 +58,7 @@ Config key:
           tabs_enabled=tabs_enabled
           callback=callback
           nocatselect=nocatselect
+          cat=cat
```

Comment:
I think we should use:

```
category=category
```

And then add `category` to the list of Params at the top.